### PR TITLE
Responsive conversations layout

### PR DESCRIPTION
**Pull request by zephraph**
Changes:
- Redirect to first conversation when `/conversations` opened on desktop
- Added
- Refactored inbox header to be responsive
- Refactored layout to be responsive
- Added details stub

Full view (details panel not yet implemented)

image

md view

image

Still quite a bit of work to do so expect a lot of fast follows.

### CODE REVIEW

**Comment by artsyit** [comment]
:rocket: PR was released in [`v26.36.0`](https://github.com/artsy/reaction/releases/tag/v26.36.0) :rocket:

File: src/Apps/Conversation/Components/Conversation.tsx [hunk 1]
```diff
@@ -124,35 +113,23 @@ export interface ConversationProps {
...
-                ? i.item.id
-                : idx
-            }
-          />
-        ))}
-        {groupMessages(conversation.messages.edges.map(edge => edge.node)).map(
-          (messageGroup, groupIndex) => {
+    <Flex flexDirection="column" width="100%">
+      <Box>
+        <Spacer mt="45px" />
```
Comment (damassi) [inline src/Apps/Conversation/Components/Conversation.tsx:118]: @zephraph wouldn't this be better if we did `mt={4}` versus breaking our design system? These values shouldn't be to comp so much as they should be to the closest value in the system. 

Thoughts?

File: src/Apps/Conversation/ConversationApp.tsx [hunk 2]
```diff
@@ -1,48 +1,102 @@
...
+  const conversation = me.conversationsConnection.edges[0]?.node
+
+  useEffect(() => {
+    setWidth(getViewWidth())
+    const listenForResize = debounce(() => {
+      setWidth(getViewWidth())
+    })
+    window.addEventListener("resize", listenForResize)
+    return () => window.removeEventListener("resize", listenForResize)
+  }, [])
```
Comment (damassi) [inline src/Apps/Conversation/ConversationApp.tsx:45]: Maybe https://reactjs.org/docs/hooks-reference.html#uselayouteffect ?

File: src/Apps/Conversation/Components/Conversation.tsx (hunk 1 above)
Comment (zephraph) [reply src/Apps/Conversation/Components/Conversation.tsx:118]: This is a buffer to move the conversation body far enough from under a fixed header to be visible. We're moving pretty fast on this and it's not all ideal at the moment. Will follow up.

File: src/Apps/Conversation/ConversationApp.tsx (hunk 2 above)
Comment (zephraph) [reply src/Apps/Conversation/ConversationApp.tsx:45]: > However, we recommend starting with useEffect first and only trying useLayoutEffect if that causes a problem.

File: src/Apps/Conversation/Components/Conversation.tsx (hunk 1 above)
Comment (damassi) [reply src/Apps/Conversation/Components/Conversation.tsx:118]: We should just add a `5` to our space scale, this constantly comes up 😄 I'll PR and ping design

**Pull request merged**